app/frontend/packs/nationalities-component.js

```diff
@@ -0,0 +1,117 @@
+const nationalitiesComponent = () => {
+  const pageHasErrors = document.querySelector(".govuk-error-summary");
+  if (pageHasErrors) return;
+
+  const secondSelect = document.getElementById(
+    "candidate-interface-nationalities-form-other-nationality2-field"
+  );
+
+  const thirdSelect = document.getElementById(
+    "candidate-interface-nationalities-form-other-nationality3-field"
+  );
+
+  const secondFormLabel = document.querySelector(
+    "[for=candidate-interface-nationalities-form-other-nationality2-field]"
+  );
+  const thirdFormLabel = document.querySelector(
+    "[for=candidate-interface-nationalities-form-other-nationality3-field]"
+  );
+
+  let addNationalityButton = null;
+
+  addRemoveLink(secondSelect.id);
+
+  addRemoveLink(thirdSelect.id);
+
+  addAddNationalityButton(
+    "#candidate-interface-nationalities-form-other-other-conditional"
+  );
+
+  hideSection(secondSelect.id);
+
+  hideSection(thirdSelect.id);
+
+  function addRemoveLink(selector) {
+    const labelEl = document.querySelector(`[for=${selector}]`);
+    const removeLink = document.createElement("a");
+    removeLink.innerHTML = "Remove";
+    removeLink.classList.add("govuk-link", "app-nationality-remove-link");
+
+    // This has to be a link and not a button as the govuk-link class requires an
+    // href to apply  it's styling
+    removeLink.href = "#";
+    labelEl.appendChild(removeLink);
+
+    addNthNationalityHiddenSpan(removeLink, selector);
```

Comment:
You shouldn't need to pass in the `selector` here either as it's only used to decide which content to use. You could pass a number instead (2 for the second label):

```suggestion
    addNthNationalityHiddenSpan(removeLink, 2);
```